recipes/alm/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "ALM" %}
+{% set version = "1.0.2" %}
+{% set sha256 = "a3ee1b25a897ec8bd576dd00c948657c4faefbd42f2616bf5ad2ce79a3d8cda4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jochym/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - use_external_spglib.patch
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - eigen
+  host:
+    - python
+    - blas 1.1 openblas  # [unix]
```

Comment:
I think what you are looking for here is `- blas * {{ blas_impl }}`. Furthermore, the '# [unix]' tag is redundant because of the `skip` line.

If it does not build, please revert and consider this comment closed.

Reply:
Where is a doc about it. I have tried `- blas * {{ openblas }}` but this did not build. Should it be `- blas 1.1 {{ openblas }}` ?

Reply:
https://conda-forge.org/docs/blas.html

Reply:
But that talks only on conda install forms not the meta.yaml syntax? These are fairly standard packages. What I need is info on what the package which needs blas/lapack should use if it needs any implementation of them and what if it needs specyfic variant (e.g. openblas).